code/src/Components/App.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import data from '../data.json';
+import stretchGoal from '../stretch-goal.json';
+import { Album } from './Album';
+import { Heading } from './Heading';
+import { Playlist } from './Playlist';
+import './App.css';
+
+//const albumArray = albums.items;
+const albumArray = data.albums.items;
+
+//creates a new array of playlist in stretch goal json
+const playlistArray = stretchGoal.playlists.items;
+
+export const App = () => {
+	return (
+		<>
+			<header className="heading">
+				<Heading />
+			</header>
```

Comment:
Maybe the `<header>` should be in the `<Heading/>` component?

Reply:
True, have moved the header tag to the heading component! 👍